lib/entrypoint.sh

```diff
@@ -2,10 +2,10 @@
 
 set -e
 
-if ["${INPUT_BUNDLE}" == "true"]
+if [ "${INPUT_BUNDLE}" == "true" ]
 then
-  bundle install --jobs 4 --retry 3
-elif ["${INPUT_VERSION}" != ""]
+  bundle install --deployment --jobs 4 --retry 3
```

Comment:
Couldn't we just do this instead? 
```suggestion
  bundle install --path vendor/bundle --jobs 4 --retry 3
```